news/setuptools.vendor.rst

```diff
@@ -1 +1 @@
-Update pkg_resources (via setuptools) to 65.6.3
+Upgrade setuptools to 67.6.1
```

Comment:
Is this change  intentional? ("Update pkg_resources (via setuptools)" becoming "Upgrade setuptools")

Reply:
I think it's a consequence of the automation handling things slightly differently than the manual upgrade I'd done earlier!

Reply:
That's what I assume - I certainly didn't do anything explicit to cause this to happen.